data/categories.txt

```diff
@@ -647,7 +647,7 @@ cs:peníze
 da:3penge
 nl:geld
 fr:argent
-de:Geld
+de:Geld|Bank|Bankfiliale
```

Comment:
"amenity-bank|@bank" means that amenity-bank _inherits_ synonyms from "@bank" declared above.
Please, check that Bank|Bankfiliale are not already defined in "@bank" to avoid useless copy-paste.